src/main/java/org/itsallcode/openfasttrace/importer/markdown/MarkdownImporter.java

```diff
@@ -38,87 +41,112 @@
+        transition(DESCRIPTION, SPEC_ITEM  , MdPattern.ID         , () -> {endDescription(); beginItem();}     ),
+        transition(DESCRIPTION, TITLE      , MdPattern.TITLE      , () -> {endDescription(); endItem();}       ),
+        transition(DESCRIPTION, RATIONALE  , MdPattern.RATIONALE  , () -> {endDescription(); beginRationale();}),
+        transition(DESCRIPTION, COMMENT    , MdPattern.COMMENT    , () -> {endDescription(); beginComment();}  ),
+        transition(DESCRIPTION, COVERS     , MdPattern.COVERS     , this::endDescription                       ),
+        transition(DESCRIPTION, DEPENDS    , MdPattern.DEPENDS    , this::endDescription                       ),
+        transition(DESCRIPTION, NEEDS      , MdPattern.NEEDS_INT  , () -> {endDescription(); addNeeds();}      ),
+        transition(DESCRIPTION, NEEDS      , MdPattern.NEEDS      , () -> endDescription()                     ),
+        transition(DESCRIPTION, DESCRIPTION, MdPattern.EVERYTHING , this::appendDescription                    ),
+        transition(DESCRIPTION, TAGS       , MdPattern.TAGS       , () -> {}                                   ),
+
     
-        transition(State.RATIONALE  , State.SPEC_ITEM  , MdPattern.ID         , () -> {endRationale(); beginItem();}       ),
-        transition(State.RATIONALE  , State.TITLE      , MdPattern.TITLE      , () -> {endRationale(); endItem();}         ),
-        transition(State.RATIONALE  , State.COMMENT    , MdPattern.COMMENT    , () -> {endRationale(); beginComment();}    ),
-        transition(State.RATIONALE  , State.COVERS     , MdPattern.COVERS     , this::endRationale                         ),
-        transition(State.RATIONALE  , State.DEPENDS    , MdPattern.DEPENDS    , this::endRationale                         ),
-        transition(State.RATIONALE  , State.NEEDS      , MdPattern.NEEDS_INT  , () -> {endRationale(); addNeeds();}        ),
-        transition(State.RATIONALE  , State.NEEDS      , MdPattern.NEEDS      , () -> endRationale()                       ),
-        transition(State.RATIONALE  , State.RATIONALE  , MdPattern.EVERYTHING , this::appendRationale                      ),
+        transition(RATIONALE  , SPEC_ITEM  , MdPattern.ID         , () -> {endRationale(); beginItem();}       ),
+        transition(RATIONALE  , TITLE      , MdPattern.TITLE      , () -> {endRationale(); endItem();}         ),
+        transition(RATIONALE  , COMMENT    , MdPattern.COMMENT    , () -> {endRationale(); beginComment();}    ),
+        transition(RATIONALE  , COVERS     , MdPattern.COVERS     , this::endRationale                         ),
+        transition(RATIONALE  , DEPENDS    , MdPattern.DEPENDS    , this::endRationale                         ),
+        transition(RATIONALE  , NEEDS      , MdPattern.NEEDS_INT  , () -> {endRationale(); addNeeds();}        ),
+        transition(RATIONALE  , NEEDS      , MdPattern.NEEDS      , () -> endRationale()                       ),
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)